nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -130,6 +198,19 @@
   implicit-imports:
     - depends:
         - "cv2.cv2"
+        - "numpy"
+        - "numpy.core"
+
```

Comment:
I will try and tweak auto-format for Yaml to enfore a separation of lines, probably on the ASCII level...